Automattic/wp-calypso
# A4A > Referrals: Redirect to client pages if the current user is a client

Resolves https://github.com/Automattic/automattic-for-agencies-dev/issues/874

## Proposed Changes

* This PR handles redirection to the client page if the requesting user is a client.

## Why are these changes being made?


* To improve the client login experiences.

## Testing Instructions



* Switch to this branch > Start the server
* Patch D157644-code
* Login as an agency > Visit http://agencies.localhost:3000/ and verify that you are redirected to the `overview` page
* Login as a client > Visit http://agencies.localhost:3000/ and verify that you are redirected to the `client/subscriptions` page
* Refresh the page and verify you are still able to view the subscription page.

## Pre-merge Checklist
